src/transformers/tokenization_utils_base.py

```diff
@@ -175,6 +175,23 @@ class TokenSpan(NamedTuple):
     end: int
 
 
+def to_py_obj(obj):
+    """
+    Convert a TensorFlow tensor, PyTorch tensor, Numpy array or python list
+    in a python list.
```

Comment:
```suggestion
    to a python list.
```

maybe also type hint -> List